scripts/plot_network.py

```diff
@@ -259,25 +258,31 @@ def split_costs(n):
 
     set_plot_style()
 
-    paths, config, wildcards, logs, out = retrieve_snakemake_keys(snakemake)
+    paths, config, wildcards, logs, out = (
+        snakemake.input,
+        snakemake.config,
+        snakemake.wildcards,
+        snakemake.log,
+        snakemake.output,
+    )
 
-    map_figsize = config['map']['figsize']
-    map_boundaries = config['map']['boundaries']
+    map_figsize = config["plotting"]['map']['figsize']
+    map_boundaries = config["plotting"]['map']['boundaries']
 
     n = load_network_for_plots(paths.network, paths.tech_costs, config)
 
     scenario_opts = wildcards.opts.split('-')
 
     fig, ax = plt.subplots(figsize=map_figsize, subplot_kw={"projection": ccrs.PlateCarree()})
-    plot_map(n, ax, wildcards.attr, config)
+    plot_map(n, ax, wildcards.attr, config["plotting"])
```

Comment:
with the non-optional `opts` `kwarg`, ordering has to be fixed... could you check it?